lib/Auth/Process/AddAdditionalAttributesAfterMfa.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimpleSAML\Module\authswitcher\Auth\Process;
+
+use SimpleSAML\Configuration;
+use SimpleSAML\Module\authswitcher\AuthSwitcher;
+
+class AddAdditionalAttributesAfterMfa extends \SimpleSAML\Auth\ProcessingFilter
+{
+    private $customAttrs;
+
+    public function __construct($config, $reserved)
+    {
+        parent::__construct($config, $reserved);
+
+        $config = Configuration::loadFromArray($config['config']);
+        $this->customAttrs = $config->getArray('custom_attrs', []);
```

Comment:
```suggestion
        $this->customAttrs = $config->getArray('custom_attrs');
```

You can make this a required config option, without it this filter does nothing.